config/wpcalypso.json

```diff
@@ -117,6 +117,15 @@
 		"upgrades/precancellation-chat": true,
 		"upgrades/presale-chat": true,
 		"woocommerce/extension-dashboard": true,
+		"woocommerce/extension-extensions": true,
+		"woocommerce/extension-orders": true,
+		"woocommerce/extension-products": true,
+		"woocommerce/extension-products-import": true,
+		"woocommerce/extension-promotions": true,
+		"woocommerce/extension-settings": true,
+		"woocommerce/extension-settings-checkout": true,
+		"woocommerce/extension-settings-shipping": true,
+		"woocommerce/extension-settings-tax": true,
 		"woocommerce/extension-stats": true,
```

Comment:
Done in https://github.com/Automattic/wp-calypso/pull/13634/commits/9161781869bc8ef0ca5b6518a2d5de6df71d44e6

cc @alisterscott